src/lxc/cgroups/cgfsng.c

```diff
@@ -503,26 +501,17 @@ static bool cg_legacy_filter_and_set_cpus(char *path, bool am_initialized)
 	bret = true;
 
 on_error:
```

Comment:
I think all users of this label could just `return false` at this point, similar for the `on_success` label.

Reply:
Agreed.